.bazelrc

```diff
@@ -14,7 +14,7 @@ build:remote --remote_timeout=7200
 # ========================================
 
 # Enable libc++ and C++20 by default.
-build --config=libc++20
+build:linux --config=libc++20
```

Comment:
why disable on macos? we have c++20 code already I think.

Reply:
```
clang: error: invalid linker name in argument '-fuse-ld=lld'
```

Reply:
you should use clang's lld on macos? what linker are you using?

Reply:
I think the default one setting xcode.

> I won't encourage anyone use proxy on macos for production, all these lines make that easier to develop on macos.

Reply:
I have no problems with allowing macos development. Just want to make sure we're not breaking release on linux.

Reply:
I do my best to make it won't happen, and there's a long time before next release, so I think it's pretty safe.